release_notes/ose_3_2_release_notes.adoc

```diff
@@ -0,0 +1,736 @@
+they were executed on the server and another user edited at the same time.
+- Users are now warned when trying to import a non-existent tag in `oc
+import-image`.
+- Singular pods are now shown in `oc status` output.
+- Router fixes:
+** More information s now shown from the router reload command in the router logs.
+** Routes that changed at the same time could compete for being exposed if they
+were in different namespaces. The check for which route gets exposed has been
+made predictable.
+** The health check is now used when restarting the router to ensure the new
+process is correctly running before continuing.
+- Better errors are displayed in the web console when JavaScript is disabled.
+- Failed deployments now update the status of the deployment configuration more
+rapidly, reducing the time before the old deployment is scaled back up.
+- Persistent volume claims (PVCs) are no longer blocked by the default SCC policy
+for users.
+- Continue to support host ports on `oadm router`. Administrators can disable them
+with `--host-ports=false` when `--host-network=false` is also set.
+- Events are now emitted when the cancellation of a deployment fails.
+- When invoking a binary build, retry if the input image stream tag does not exist
+yet (because it may be in the process of being imported).
+- Fixed a race condition in Kubernetes where endpoints might be partially updated
+(only have some pods) when the controller is restarted.
+- Docker containers do not allow CPU quota less than `10m`, so set the minimum
+value.
+- Do not sync `*DaemonSet*` objects that match all pods.
+- The `oc new-build` command no longer fails when creating a binary build on a Git
+repository that does not have an upstream remote set.
+- Fixed a race condition between scaled up routers where some changes might be
+ignored.
+- Enable the etcd watch cache for Kubernetes resources, reducing memory use and
+duplicate watches.
+- Change the `*RunOnce*` pod duration restrictor to act as a limit instead of
+override.
+- Guarantee partially completed builds are cleaned up when cancelled.
+- Check `*claimRef*` UID when processing a recycled persistent volume (PV) to
+prevent races.
+- The `*ProjectRequestLimit*` plug-in now ignores projects in terminating state.
+- The `*ConfigMap*` volume is now readable as non-root.
+- The *system:image-auditor* role has been added for managing the image registry.
+- Dynamic volume provisioning can now be disabled.
+- Deployment pods should now be cancelled when deployments are cancelled in all
+cases.
+- The deployer controller should now ensure deployments that are cancelled can not
+become completed.
+- Concurrent deployer pod creation is now prevented.
+- Fixed an issue where a pod would never terminate if the registry it pulls images
+from was unavailable.
+- Fixed precision of CPU to millicore and memory to Mi in the UI.
+- The HAProxy router should now obfuscate the pod IP in when using cookies for
+session affinity.
+
```

Comment:
I suppose the above section was generated from PR (or BZ or Jira) titles, and as such, can be expected to be in different styles.  Ah well...